modules/internal/ChapelError.chpl

```diff
@@ -340,6 +340,12 @@ module ChapelError {
     return chpl_fix_thrown_error(_to_unmanaged(err));
   }
 
+  pragma "no doc"
+  pragma "insert line file info"
+  proc chpl_fix_thrown_error(type Error) {
```

Comment:
Can we rename this argument to errorType, to distinguish from the global type named Error?

Reply:
the other args are named `err` and have type `Error`

Reply:
(you could rename this argument to `err` and that'd be reasonable)